R/plumb-globals.R

```diff
@@ -32,37 +32,28 @@ plumbOneGlobal <- function(fields, argument){
            fields$info$termsOfService <- def
          },
          apiContact={
+           if (grepl("^list\\(", def)) {
+             def <- eval(parse(text = def))
+           }
            fields$info$contact <- def
          },
          apiLicense={
+           if (grepl("^list\\(", def)) {
+             def <- eval(parse(text = def))
+           }
            fields$info$license <- def
          },
          apiVersion={
            fields$info$version <- def
          },
-         apiHost={
-           fields$host <- def
-         },
-         apiBasePath={
-           fields$basePath <- def
-         },
-         apiSchemes={
-           fields$schemes <- strsplit(def, split="\\s+")[[1]]
-         },
-         apiConsumes={
-           fields$consumes <- strsplit(def, split="\\s+")[[1]]
-         },
-         apiProduces={
-           fields$produces <- strsplit(def, split="\\s+")[[1]]
-         },
          apiTag={
            tagMat <- stri_match(def, regex="^\\s*(\\w+)\\s+(\\S.+)\\s*$")
            name <- tagMat[1,2]
            description <- tagMat[1,3]
-           if(!is.null(fields$tags) && name %in% fields$tags$name) {
+           if(!is.null(fields$tags) && name %in% sapply(fields$tags, "[[", "name")) {
```

Comment:
I try to avoid `sapply` at all costs due to the inconsistent simplifying behavior.  If we know it's a vector, we should use `vapply`. If it's a vector, but may not have a `name` in each item, then we can `unlist(lapply())`

```suggestion
           if(!is.null(fields$tags) && name %in% unlist(lapply(fields$tags, "[[", "name"))) {
```